samples/wormhole/ping-pong/sample.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Using Wormhole to send messages between distributed sites
+component: Wormhole
+reviewed: 2017-05-26
+related:
+ - nservicebus/wormhole
+---
+
+## Prerequisistes
+
+Open the solution in VisualStudio running as Administrator because the gateways need to register the listening URL. Alternatively use add the registrations manually: 
+
+```
+netsh add urlacl url=http://+:7777/Gateway.SiteA/ user=DOMAIN\user
+netsh add urlacl url=http://+:7777/Gateway.SiteB/ user=DOMAIN\user
+``` 
+
+## Running the project
+
+ 1. Start the projects: Client, Server and both gateways (right-click on the project, select the `Debug > Start new instance` option).
```

Comment:
I've copied this from other samples. But you're right, I'll remove this. It is redundant.